app/components/threads_button/threads_button.tsx

```diff
@@ -54,33 +59,44 @@ const ThreadsButton = ({currentChannelId, groupUnreadsSeparately, onlyUnreads, u
     const customStyles = getStyleSheet(theme);
 
     const handlePress = useCallback(preventDoubleTap(() => {
-        switchToGlobalThreads(serverUrl);
+        if (onPress) {
```

Comment:
Unless we expect onPress to be constant between renders, we should add it to the dependencies, right?